### ISSUE TITLE
mpi4.1: allow MPI_COMM_NULL to be passed to MPI_Comm_get_name

### ISSUE THREAD

**Issue by hppritcha** [open-mpi/ompi#12079]
See item 18 in chapter B 1.2 in MPI 4.1 spec

**Comment by dalcinl** [comment]
@hppritcha @edgargabriel  What about `MPI_Datatype` and `MPI_Win`? 
Same new rules apply. Did you miss it from the MPI 4.1 changelog?

**Comment by edgargabriel** [comment]
@dalcinl yes, we probably missed it. I will work on this in the next few days. Thanks!
